client/app/templates/components/advisory-message-bar.hbs

```diff
@@ -0,0 +1,5 @@
+<div class="advisory-message">
```

Comment:
Maybe check if `this.message` is truthy and conditionally render the `div` based on that? That way we never accidentally render an empty bar.

Reply:
I don't think we would accidentally render an empty div as there is a check in the application template looking for the truthy value of the feature flag.

Were you thinking of something like:
```
{{#if this.message}}
  <div class="advisory-message">
  // ... code
```